wordpress-mobile/WordPress-Android
# [Jetpack Full Plugin] Update texts in card and dialog

Fixes #18117 

- Change the card text to be more concise and take up less space.
- Change the overlay text to use the site URL and not the site name.

https://user-images.githubusercontent.com/5091503/225712091-23a3c50b-846d-4d8e-810b-711cc8f2b42e.mp4

To test:
Set up a self-hosted site that has only individual Jetpack plugins enabled and connect Jetpack to a WP.com account.

1. Install Jetpack app
2. Log in with the same WP.com account
2. Restart the app (so the site appears in the site list)
2. Select the self-hosted connected site from the site list
2. The dialog asking to install the full plugin should appear AND/OR the card on the Dashboard
2. **Verify** the card text match the new expectations
3. **Verify** the overlay text match the new expectations

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

4. What automated tests I added (or what prevented me from doing so)
Only fixed tests due to usage of URL instead of a site name. Otherwise, it was just visual changes.

PR submission checklist:
